packages/core/src/html/warnings.ts

```diff
@@ -9,12 +10,14 @@ const { getVslotShorthandName } = require('./vueSlotSyntaxProcessor');
  * @param attribute An attribute that is conflicting with other attributes
  * @param attrsConflictingWith The attributes conflicting with `attribute`
  */
-function _warnConflictingAttributes(node, attribute, attrsConflictingWith) {
-  if (!(attribute in node.attribs)) {
+function _warnConflictingAttributes(node: DomElement, attribute: string, attrsConflictingWith: string[]) {
+  if ((!node.attribs) || (!(attribute in node.attribs))) {
     return;
   }
+
+  const nodeAttribs = node.attribs;
   attrsConflictingWith.forEach((conflictingAttr) => {
-    if (conflictingAttr in node.attribs) {
+    if (conflictingAttr in nodeAttribs) {
```

Comment:
Why is `node.attribs` extracted as `nodeAttribs` here?

Reply:
The linter complains that `node.attribs` is possibly undefined over here.